protocol075.md

```diff
@@ -505,13 +505,6 @@ data, where the gamemode is sent. It could be considered as part of that
 initial data packet, but as what's sent varies greatly depending on the
 gamemode, it is documented separately.
 
-## Territory
```

Comment:
Why remove this?

Reply:
because there is no packet `Territory` nor `Object Territory`